--- utils/constants.js
+++ utils/constants.js
@@ -1,16 +1,9 @@
 const DEFAULT_LOAN_AMOUNT = 412110.84;
-<<<<<<< HEAD
-const EXPORT_FILENAME = "Mortgage_Payments_History.xlsx";
-=======
 const EXPORT_FILE_NAME = "Mortgage_Payments_History.xlsx";
->>>>>>> origin/main
 const EXPORT_SHEET_NAME = "Payment History";

 module.exports = {
     DEFAULT_LOAN_AMOUNT,
-<<<<<<< HEAD
-    EXPORT_FILENAME,
-=======
     EXPORT_FILE_NAME,
->>>>>>> origin/main
     EXPORT_SHEET_NAME
 };
